clang/lib/Driver/Driver.cpp

```diff
@@ -674,6 +674,25 @@ static bool isValidSYCLTriple(llvm::Triple T) {
   return true;
 }
 
+static llvm::Triple completeSYCLTriple(llvm::Triple T) {
```

Comment:
Probably you can describe what the function does at high-level and intersperse a few comments about the logic.
I am unsure that everybody knows the world-famous `XCL_EMULATION_MODE` environment variable...